### PR TITLE
Minimal UI testing

### PR DESCRIPTION
**Pull request by karosas**
Hi there, decided to contribute a little and noticed you don't have any tests (Shame) so I wrote a few UI tests using Espresso.
- Added couple of dependencies for android test app.
- Added couple app's build.gradle entries to configure test app with espresso and enable coverage output
- Coverage report is in /app/build/reports/coverage/debug

If you don't need any of this, just ignore it :)

P.S. I'm super new to contributing to OS, so if I made some mistakes related to git then please let me know.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -149,6 +156,14 @@ dependencies {
     compile('com.crashlytics.sdk.android:crashlytics:2.6.2@aar') {
         transitive = true;
     }
+
+    testCompile "org.mockito:mockito-core:$mockito"
+    androidTestCompile "com.android.support.test.espresso:espresso-core:$espresso"
+    androidTestCompile("com.android.support.test.espresso:espresso-intents:$espresso")
```
Comment (vycius) [inline app/build.gradle:162]: Let's keep same style for all dependencies. Let's change this to 

```
androidTestCompile "com.android.support.test.espresso:espresso-intents:$espresso"
```

File: app/build.gradle [hunk 2]
```diff
@@ -149,6 +156,14 @@ dependencies {
...
+    androidTestCompile 'com.android.support.test:rules:0.5'
```
Comment (vycius) [inline app/build.gradle:163]: Change to:

```
androidTestCompile "com.android.support.test:rules:$testRunner"
```

File: app/build.gradle [hunk 3]
```diff
@@ -149,6 +156,14 @@ dependencies {
...
+    androidTestCompile "com.android.support.test:runner:$testRunner"
+    androidTestCompile "com.android.support:appcompat-v7:$supportLibraryVersion"
```
Comment (vycius) [inline app/build.gradle:165]: `com.android.support:appcompat-v7` is already added as `compile` dependency

File: app/build.gradle [hunk 4]
```diff
@@ -149,6 +156,14 @@ dependencies {
...
+    androidTestCompile "com.android.support:support-v4:24.1.1" // 23.4.0 causes dependency error
```
Comment (vycius) [inline app/build.gradle:166]: @inetaa I will create PR to upgrade to newest Android version and we will no longer need this line.

**Comment by vycius** [comment]
Could you rebase this on top of `vilnius:master`, not on `fix-beta-issues` ?

**Comment by vycius** [comment]
👍 I love this tests UI tests initiative.

**Comment by karosas** [comment]
Screwed my local fork, reforked and sent new pull request

**Comment by vycius** [comment]
New pull request created https://github.com/vilnius/tvarkau-vilniu/pull/70

**Pull request closed without merging**